Scratch_GUI/Scratch_Start.py

```diff
@@ -283,7 +287,10 @@ def start_programming(self, event):
 			else: #BrickPi+, Kickstarter and Advanced,
 				program = "/home/pi/Dexter/BrickPi+/Software/BrickPi_Scratch/BrickPiScratch.py"
 		elif user_selection.find('GoPiGo') >= 0:
-			program = "/home/pi/Dexter/GoPiGo/Software/Scratch/GoPiGoScratch.py"
+			if autodetect().find("GoPiGo3") >= 0:
```

Comment:
Elegant way of dealing with the string finding issue of finding "GoPiGo" inside "GoPiGo3"